### PR TITLE
Get working with latest relayer

### PR DESCRIPTION
**Pull request by michaelfig**
Upgrade SDK and get working with [latest relayer](https://github.com/agoric-labs/relayer/tree/mfig/nchainz-agoric).  See `scripts/agoric-solo` in that repo.

### CODE REVIEW

**Comment by michaelfig** [comment]
For reviewers, please turn on "ignore whitespace" to see what I actually changed.

**Review by FUDCo — APPROVED** [review]
Looks reasonable to me, though take that with a grain of salt given my relative lack of familiarity with Go and with the problem space.

**Review by warner — APPROVED** [review]
I didn't see anything wrong with it

File: packages/cosmic-swingset/bin/ag-nchainz [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      # Now wire into the chain.
+      gci=`$thisdir/../calc-gci.js $n0d/config/genesis.json`
+      rpcport=`$thisdir/../calc-rpcport.js $n0d/config/config.toml`
+
+      $SOLO set-gci-ingress --chainID="$chainid" "$gci" "$rpcport"
+
+      DEBUG=agoric $SOLO start --role=two_client
+    ) >>nchainz/logs/$log 2>&1 &
+  done
+  sleep 2
```
Comment (kriskowal) [inline packages/cosmic-swingset/bin/ag-nchainz:102]: If we control the implementation of $SOLO, we can do better than wait time. Like, we could arrange for a temporary fifo and pass it as an argument. The solo would be obliged to write a line after it is open for business. You would `read -r _ < tmp.fifo` here. There is probably a better way. This technique would resume early if the kernel interrupts the read. Not sure I can think of another way without a spin lock.

**Review by kriskowal — APPROVED** [review]

**Pull request merged**